.github/workflows/analysis.yml

```diff
@@ -42,11 +42,16 @@ jobs:
     needs: run_linters
     steps:
       - uses: actions/checkout@v2
+
       - run:  sudo apt-get update
-      - name: Install C++ compiler and libraries
-        run:  sudo apt-get install python3-setuptools $(./scripts/list-build-dependencies.sh -m apt -c gcc)
-      - name: Install scan-build (Python version)
-        run:  sudo pip3 install scan-build beautifulsoup4 html5lib
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install clang-tools python3-setuptools \
+            $(cat .github/packages/ubuntu-20.04-apt.txt)
```

Comment:
To wrap long YAML values across multiple lines and eat the new-line and indentation, use:

``` yaml
run: >
    One big long
    run-on line.
```

Reply:
I wanted to have second line indented for readability and:
```yaml
run: >
  sudo apt-get install clang-tools python3-setuptools
    $(cat .github/packages/ubuntu-20.04-apt.txt)
```
… doesn't work (newline is being injected in second line?)

I found this comprehensive answer about yaml syntax: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210 and ended up doing this:

```yaml
run: sudo apt-get install clang-tools python3-setuptools
                          $(cat .github/packages/ubuntu-20.04-apt.txt)
```
(which, to my surprise, worked)

Reply:
Indeed, that's also a suprise to me. Folded type (http://www.yaml.org/spec/1.2/spec.html#id2796251) makes the most logical sense, but I suppose it didn't like the uneven indentation. 

Hey, if it works and achieves the goal without any extra syntax, then its the best of both worlds!